completor/prepare_outputs.py

```diff
@@ -91,42 +91,121 @@ def dataframe_tostring(
     if columns[0] != "--":
         # then add first column
         df_temp = add_columns_first_last(df_temp, add_first=True, add_last=False)
-    # Add single quotes around well names in output file
+
+    # Add single quotes around well names in an output file.
     if Headers.WELL in df_temp.columns:
         df_temp[Headers.WELL] = "'" + df_temp[Headers.WELL].astype(str) + "'"
-    output_string = df_temp.to_string(index=False, justify="justify", header=header)
+
+    formatters: Mapping[Any, Any] = {}
     if format_column:
-        if formatters is None:
-            formatters = {
-                Headers.STRENGTH: "{:.10g}".format,
-                Headers.SCALE_FACTOR: "{:.10g}".format,
-                Headers.ROUGHNESS: "{:.10g}".format,
-                Headers.CONNECTION_FACTOR: "{:.10g}".format,
-                Headers.FORMATION_PERMEABILITY_THICKNESS: "{:.10g}".format,
-                Headers.MEASURED_DEPTH: "{:.3f}".format,
-                Headers.TRUE_VERTICAL_DEPTH: "{:.3f}".format,
-                Headers.START_MEASURED_DEPTH: "{:.3f}".format,
-                Headers.END_MEASURED_DEPTH: "{:.3f}".format,
-                Headers.FLOW_COEFFICIENT: "{:.10g}".format,
-                Headers.FLOW_CROSS_SECTIONAL_AREA: "{:.3e}".format,
-                Headers.OIL_FLOW_CROSS_SECTIONAL_AREA: "{:.3e}".format,
-                Headers.GAS_FLOW_CROSS_SECTIONAL_AREA: "{:.3e}".format,
-                Headers.WATER_FLOW_CROSS_SECTIONAL_AREA: "{:.3e}".format,
-                Headers.MAX_FLOW_CROSS_SECTIONAL_AREA: "{:.3e}".format,
-                Headers.DEFAULTS: "{:.10s}".format,
-                Headers.WATER_HOLDUP_FRACTION_LOW_CUTOFF: "{:.10g}".format,
-                Headers.WATER_HOLDUP_FRACTION_HIGH_CUTOFF: "{:.10g}".format,
-                Headers.GAS_HOLDUP_FRACTION_LOW_CUTOFF: "{:.10g}".format,
-                Headers.GAS_HOLDUP_FRACTION_HIGH_CUTOFF: "{:.10g}".format,
-                Headers.ALPHA_MAIN: "{:.10g}".format,
-                Headers.ALPHA_PILOT: "{:.10g}".format,
-            }
-        try:
-            output_string = df_temp.to_string(index=False, justify="justify", formatters=formatters, header=header)
-        except ValueError:
-            pass
+        formatters = {
+            Headers.STRENGTH: "{:.10g}".format,
+            Headers.SCALE_FACTOR: "{:.10g}".format,
+            Headers.ROUGHNESS: "{:.10g}".format,
+            Headers.CONNECTION_FACTOR: "{:.10g}".format,
+            "CONNECTION_FACTOR": "{:.10g}".format,
```

Comment:
Add header variations to Headers in constants.py and use them from there?
Maybe add a default format to the headers as well and get them from there?

Reply:
See: https://github.com/equinor/completor/issues/168